conversational-client/src/app/app.component.html

```diff
@@ -1,8 +1,15 @@
-<mat-toolbar color="primary">
-  <span>Conversational Client</span>
-</mat-toolbar>
-<app-speech-recognizer
-  (newDialogLineEvent)="handleNewLineOfDialog($event)"
-  (transcriptDowloadEvent)="handleTranscriptDowloadEvent($event)"
->
-</app-speech-recognizer>
+<mat-sidenav-container>
```

Comment:
nit: consider a dropdown instead of a side drawer, which is generally used for navigation

Reply:
will stick with side drawer for now, I am not sure yet if this will actually feel more like a navigation than anything else. but need to play with a few more usecases first.